### PR TITLE
Feature/merge view only and normal db models

### PR DESCRIPTION
**Pull request by BrianCorbin**
### Tests
- [x] View Only Account Import
- [x] View Only Account Sync Process
- [x] VO Assigned Subaddress Creation

### CODE REVIEW

File: full-service/src/service/account.rs
Comment (Shramp) [inline full-service/src/service/account.rs:71]: I think these two errors still need to be implimented?

File: full-service/src/util/encoding_helpers.rs
Comment (Shramp) [inline full-service/src/util/encoding_helpers.rs:7]: stoked to see that these helpers are being used!

**Review by Shramp — CHANGES_REQUESTED** [review]
Diff is a bit large for deep understanding of everything happening here, but most of it makes sense to me and looks like. Just a few minor questions/requests

**Review by christian-oudard — APPROVED** [review]

File: full-service/migrations/2022-06-13-204000_api_v3/up.sql
Comment (christian-oudard) [reply full-service/migrations/2022-06-13-204000_api_v3/up.sql:2]: This is starting the new 2.0 release series, and it seems prohibitively difficult at this time to do the data migrations necessary to untangle the separate view-only accounts. So, we decided to wipe the current migration table, and start afresh, requiring people to re-import.

**Review by Shramp — APPROVED** [review]
LGTM once merge conflicts are resolved

**Pull request merged**
